src/Ilios/CliBundle/Resources/config/services.yml

```diff
@@ -0,0 +1 @@
+services:
```

Comment:
could be removed. but left in there for later usage. if it is still empty at the end of the next round of custom command implementations then i'll yank it.